float32 position1=0
float32 position2=180
float32 position3=180
float32 position4=90
float32 position5=0
---
bool success